Runtime/Scripts/Jobs.cs

```diff
@@ -967,6 +967,26 @@ public void Execute(int i)
         }
 #endif
     }
+
+    [BurstCompile]
+    public unsafe struct ConvertColorsRGBFloat4ToRGBAFloatJob : IJobParallelFor {
```

Comment:
@atteneder I realized I probably should have named this ConvertColorsRGBAFloatToRGBAFloatJob

Reply:
no worries. I already renamed this one (and others) for more consistent naming (will push soon)